countdown.rb

```diff
@@ -5,64 +5,115 @@
 
 require_relative 'helpers'
 
-clear
 
+a = Artii::Base.new :font => 'ogre'
+b = Artii::Base.new :font => 'catwalk'
+c = Artii::Base.new :font => 'cosmic'
+d = Artii::Base.new :font => 'invita'
```

Comment:
Are we going to use 4 different fonts to display messages? Maybe that's too much :laughing: